queries/html/injections.scm

```diff
@@ -6,7 +6,50 @@
    (quoted_attribute_value (attribute_value) @css))
  (#eq? @_attr "style"))
 
-((script_element
-  (raw_text) @javascript))
-
 (comment) @comment
+
+(element
+   (self_closing_tag
+      (tag_name) @_tagname (#eq? @_tagname "input")
+      ((attribute
+          (attribute_name) @_attr
+          (quoted_attribute_value (attribute_value) @regex)))
+       (#eq? @_attr "pattern")))
+
+(element
+   (start_tag
+      (tag_name) @_tagname (#eq? @_tagname "input")
+      ((attribute
+          (attribute_name) @_attr
+          (quoted_attribute_value (attribute_value) @regex)))
+       (#eq? @_attr "pattern")))
+
+(script_element
+   (start_tag
+      ((attribute
+           (attribute_name) @_attr (#eq? @_attr "type")
+           (quoted_attribute_value (attribute_value) @_type))
+        (#any-of? @_type "module" "application/javascript")))
```

Comment:
Might want to use a more complete list https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#textjavascript

Reply:
done thanks